CHANGES.md

```diff
@@ -1,3 +1,21 @@
+## edge-19.5.4
+
+* CLI
+  * Fixed an issue where, when Linkerd is installed with `--ha`, running
+    `linkerd upgrade` without `--ha` will disable the high availability
```

Comment:
I was a little confused about "will disable the high availability control plane", do you mean that is NOW the behavior or it was the previous behavior? Should the verbs be "when Linkerd was installed... ...would disable..."?

Reply:
Yeah, I think this should be past tense. Thanks!